src/content/plugins/split-chunks-plugin.md

```diff
@@ -369,4 +369,4 @@ module.exports = {
 };
 ```
 
- T> This will result in splitting `react` and `react-dom` into a separate chunk.
+ T> This will result in splitting `react` and `react-dom` into a separate chunk. If you're not sure what packages have been included in a chunk you may refer to [Bundle Analysis](https://webpack.js.org/guides/code-splitting/#bundle-analysis) section for details.
```

Comment:
Please use relative URLs `/guides/code-splitting/#bundle-analysis`

Reply:
@montogeek pushed the requested changes. Thanks